model_analyzer/analyzer.py

```diff
@@ -119,9 +119,17 @@ def profile(self, client, gpus):
         profiled_model_list = list(
             self._state_manager.get_state_variable(
                 'ResultManager.results').keys())
+
+        num_profiled_configs = sum([
+            len(x) for x in self._state_manager.get_state_variable(
+                'ResultManager.results').values()
+        ])
         logger.info(
-            f"Finished profiling. Obtained measurements for models: {profiled_model_list}."
-        )
+            f"Profile complete. Profiled {num_profiled_configs} configurations "
```

Comment:
example:

```
...
2021-11-19 01:28:58.440 INFO[analyzer.py:127] Profile complete. Profiled 2 configurations for models: ['add_sub']. To analyze the profile results and find the best configurations, please run `model-analyzer analyze --analysis-models add_sub`.
$ 
```

Reply:
That is one really long line. Can we add your changes to a new logger line?

Reply:
broke up into multiple calls to `logger.info()` 👍